requirements.txt

```diff
@@ -2,7 +2,7 @@ pandas
 progressbar2
 python-utils
 tensorflow == 1.12.0
-numpy
+numpy == 1.15.4
```

Comment:
Why ?

Reply:
If it's because of 1.16.1 erroring, this should be fixed now: https://github.com/bennuttall/piwheels/issues/127

Reply:
https://github.com/pandas-dev/pandas/issues/24839